src/query/query_executor.c

```diff
@@ -692,6 +692,11 @@ static int qexec_locate_agg_hentry_in_list (THREAD_ENTRY * thread_p, AGGREGATE_H
 					    AGGREGATE_HASH_KEY * key, bool * found);
 static int qexec_get_attr_default (THREAD_ENTRY * thread_p, OR_ATTRIBUTE * attr, DB_VALUE * default_val);
 
+static inline SCAN_CODE evaluate_xptr_list (THREAD_ENTRY * thread_p, XASL_STATE * xasl_state, XASL_NODE * xptr_list,
+					    SCAN_OPERATION_TYPE scan_operation_type, bool * keep_going);
+static inline SCAN_CODE evaluate_one_scanned_tuple (THREAD_ENTRY * thread_p, XASL_NODE * xasl, XASL_STATE * xasl_state,
+						    QFILE_TUPLE_RECORD * tplrec, bool * keep_going);
+
```

Comment:
Can I know why these functions are inlined? I think these functions are too long, so the inline keyword is going to be ignored.

Reply:
Even if inline is specified, it may be excluded by the compiler.